app/models/conversation.rb

```diff
@@ -59,15 +44,12 @@ def set_read(user)
     end
   end
 
-  def public?
-    false
-  end
-
   def participant_handles
-    self.participants.map{|p| p.diaspora_handle}.join(";")
+    participants.map(&:diaspora_handle).join(";")
```

Comment:
this method is called while dispatching, and the participants are probably already loaded, because he calls `subscribers` before he calls `participant_handles`.